smooth/components/component_battery.py

```diff
@@ -37,26 +37,35 @@
 
 Maximum chargeable/dischargeable energy
 ---------------------------------------
-The maximum chargeable and dischargeable energy [Wh] per timestep is dependant on
-the C-rate and the state of charge of the battery. Because of the charging losses
-defined by charging efficiency, the nominal value of the input flow must be higher
-than what is actually entering the battery in order to ensure that the battery can
-be fully charged in one time step. Due to the *inflow_conversion_factor*
-parameter (in :func:`~smooth.components.component_battery.Battery.create_oemof_model`),
-the battery will then receive the right amount.
+The maximum chargeable or dischargeable power [W] going in to or out of the battery is dependant
+on the C-rate and the capacity:
 
 .. math::
-    E_{in,max} = \\frac{min(c_{r,charge} \\cdot C \\cdot \\frac{t}{60},
-    C - SOC \\cdot C)}{\\mu_{charge}} \n
-    E_{out,max} = min(c_{r,discharge} \\cdot C \\cdot \\frac{t}{60}, SOC \\cdot C)
+    P_{charge,max} = C * c_{r,charge}
 
-* :math:`E_{in,max}` = maximum chargeable energy [Wh]
-* :math:`c_{r,charge}` = C-Rate [-/h]
+    P_{discharge,max} = C * c_{r,discharge}
+To ensure that the battery can be charged and discharged within the time frame inherently defined
+by the C-rate (1C : Full capacity can be (dis-)charged within one hour), the nominal value of
+the input-flow/output-flow between bus and battery needs to include the energy losses during the
+(dis-)charging process:
+
+.. math::
+    P_{in,max} = P_{charge,max} / \\mu_{charge}
+
+    P_{out,max} = P_{discharge,max} * \\mu_{discharge}
+Due to the inflow_conversion_factor / outflow_conversion_factor
+(in :func:`~smooth.components.component_battery.Battery.create_oemof_model`)
+the battery will then receive right amount.
+
+* :math:`P_{charge,max}` = maximum chargeable power at the battery [W]
 * :math:`C` = storage capacity [Wh]
```

Comment:
I find the use of C for capacity quite confusing, when there is as well a c_rate with a unit [C]

Could we use sth. like E_{storage,max}?

Reply:
I agree, I also find it confusing. Capacity also refers to [Ah] as well as [Wh] in literature. But in the context of batteries mainly to [Ah], so with the formula sign "E" one could prevent that confusion. 
I would then propose` E_{storage}` = storage capacity [Wh]` as the "max" could be confusing, as it has nothing to to with P_{charge,max} for example.